src/context/init.js

```diff
@@ -32,6 +33,7 @@ const AuthorizationFinder = require('../services/authorization-finder');
  */
 function initUtils(context) {
   context.addInstance('errorMessages', errorMessages);
+  context.addInstance('forestUrlGetter', forestUrlGetter);
```

Comment:
[THINKING] I'm wondering if this should not be considered as a value instead of a service on its own